packages/design-system/src/components/FormLabel/FormLabel.tsx

```diff
@@ -84,7 +87,7 @@ export class FormLabel extends React.PureComponent<
     }
 
     return (
-      <span className={classes}>
+      <span id={hintId} className={classes}>
```

Comment:
@SparrowSpit had a `<p>` tag being used in his example, but that tag still has user-agent styles on it that resulted in a jacked up layout. Using `<span>` as a short-term fix until we prove this a11y solution is solid, then I can refactor to `<p>` and style appropriately.